docs/policy_templates_data/policy_templates.json

```diff
@@ -1239,6 +1239,34 @@
           }
         ]
       }
+    },
+    "FilterLogEventsPolicy": {
+      "Description": "Gives permission to filter Log Events from any Log Group",
+      "Parameters": {
+        "LogGroupName": {
+          "Description": "Name of the Log Group"
+        }
+      },
+      "Definition": {
+        "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": [
+              "logs:FilterLogEvents"
+            ],
+            "Resource": {
+              "Fn::Sub": [
+                "arn:${AWS::Partition}:logs:${AWS::Region}:${AWS::Account}:log-group:${logGroupName}*",
```

Comment:
Do you need the "*" at the end? You can try the policy definition in your own template to figure out.

Reply:
I put it there because of the lambda invoke example I was following. I don't believe it is needed though.